extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -120,6 +120,31 @@ export class AmpAnalytics extends AMP.BaseElement {
           'config. No analytics data will be sent.');
       return Promise.resolve();
     }
+    if (this.config_['customVars'] && this.config_['customVarsMap']) {
+      this.config_['_customVars'] = this.config_['customVars'];
+      // If the vendor-provided config includes a customVarsMap, apply it as a
+      // set of params to String.replace to allow aliasing of the keys in
+      // customVars.
+      var i = 0;
+      for (const keyrule in this.config_['customVarsMap']) {
+        var replacerule = this.config_['customVarsMap'][keyrule];
+        alert ("keyrule = " + keyrule + ";replacerule = " + replacerule);
+        if (++i > 16) {
+          console./*OK*/error(this.getName_(), "More than 16 customVarsMap rules aren't allowed.  Skipping the rest");
+          break;
+        }
+        for (const key in this.config_['customVars']) {
+          var val = this.config_['customVars'][key];
+          // only allow word chars
```

Comment:
The `.replace(string, …)` form you use above actually does not use a regex for the replace operation.